### PR TITLE
6981 cms upload through azure

### PR DESCRIPTION
**Pull request by habitullence**

### CODE REVIEW

File: .env.dev [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+GA_API_EMAIL_ADDRESS="google.service.account@example.com"
+GA_PRIVATE_KEY_PATH="path/to/private.key"
+MAS_CMS_URL="http://localhost:3000/"
+PUBLIFY_HOSTNAME="blog.moneyadviceservice.org.uk"
+PUBLIFY_PORT="443"
+PUBLIFY_USE_SSL=true
+RACKSPACE_API_KEY='key'
+RACKSPACE_API_KEY='key'
+RACKSPACE_BUCKET_NAME='bucket'
+RACKSPACE_USERNAME='username'
```
Comment (tomas-stefano) [inline .env.dev]: Nice!

File: config/application.rb [hunk 2]
```diff
@@ -28,5 +28,7 @@ class Application < Rails::Application
     config.assets.precompile += %w(comfortable_mexican_sofa/admin/basic.css rsvp/rsvp.js)
 
     config.middleware.use Rack::RedirectMiddleware::Responder
+
+    config.comfy = config_for(:comfy)
```
Comment (tomas-stefano) [inline config/application.rb]: Is this line needed?

File: config/initializers/comfortable_mexican_sofa.rb [hunk 3]
```diff
@@ -32,8 +33,17 @@
   # http://rdoc.info/gems/paperclip/2.3.8/Paperclip/Storage/S3, and for
   # filesystem see: http://rdoc.info/gems/paperclip/2.3.8/Paperclip/Storage/Filesystem
   # If you are using S3 and HTTPS, pass :s3_protocol => '' to have URLs that use the protocol of the page
-  #   config.upload_file_options = {:url => '/system/:class/:id/:attachment/:style/:filename'}
-  config.upload_file_options = { styles: { cms_medium: '160x160' } }
+  config.upload_file_options = {
+    azure_credentials: {
+      access_key: <%= ENV['AZURE_ASSETS_STORAGE_CMS_ACCOUNT_KEY'] %>,
```
Comment (jongilbraith) [inline config/initializers/comfortable_mexican_sofa.rb]: Erb style tags in with your ruby?

Comment (habitullence) [inline config/initializers/comfortable_mexican_sofa.rb]: I mayyy have done a blind find and replace ;)

**Comment by habitullence** [comment]
Just updated this to resolve some merge conflicts.

**Comment by tomas-stefano** [comment]
LG :+1: :+1: The scripts that supports this environment variables are on azure-migration: https://github.com/moneyadviceservice/scripts/tree/azure-migration Doubt: Are you sure this should be merged on the master?

**Comment by habitullence** [comment]
Ahhh! I didn't realise there was an Azure branch, that makes sense.

**Pull request closed without merging**